Update bare metal deployments (#1055)
owncloud/docs-ocis: modules/ROOT/pages/depl-examples/bare-metal.adoc

=== Install and configure the Infinite Scale Binary

* Download and install the Infinite Scale binary. To get the stable binary from ownCloud, visit {ocis-downloadpage-url}/{download-type}/?sort=time&order=desc[download.owncloud.com,window=_blank], select the version and the platform of choice and copy the link of the file. Replace `<file_url>` and `<version>` in the commands accordingly.
+
--
Download the source:

[source,bash,subs="attributes+"]
----
sudo wget -P {ocis_bin} <file_url>
----
--

* Make the binary executable with:
+
[source,bash,subs="attributes+"]
----
sudo chmod a+x {ocis_bin}/ocis-<version>
----

> * Create a link from the source to the final executable named `ocis`: +

Reviewer: ```suggestion
* Create a link from the versioned ocis binary name to the final executable named `ocis`: +
```

> Note when using a link to the source, upgrading and/or testing is much easier.

Reviewer: I am puzzled. One of a sudden, the binary is called source. Do I need to compile something now?
```suggestion
Note when using a symbolic link like this, upgrading and/or testing is much easier.
```
